### DISCUSSION TITLE
How do I include the bibliography at the end of a separate file?

### DISCUSSION THREAD

**Discussion by krishaamer** [quarto-dev/quarto-cli#6454] [Q&A] [answered]
### Description

My writing project (thesis) consists of several .qmd files (mostly because it's also rendered as a website) and by default all the references will appear at the end of that particular file. I would however like to show the bibliography only in one file/page (in case of the website format) and in the case of PDF at the end of the entire document (concated from the several .qmd files). Hope my explanation makes sense. Here's a link to the actual project if it makes it clearer: https://www.greenfilter.app/research/references 

Does anyone have any advice?

Thanks
Kris

**Comment by dragonstyle** [comment, marked as answer]
One idea would be to use a `book` project instead of a `website` project. Book projects do place all the references at the end of the book in a single standalone section rather than on a per page basis. Books include a website (and more or less support all the various `website` options for customizing the appearance).

**Reply by NeginValizadegan** [reply]
I am using a book but still doesn't show reference in a separate .qmd file at the end. I created a file called References.qmd and added it to the .yml file like this:

`bibliography: references.bib`

**Reply by mcanouil** [reply]
@NeginValizadegan See <https://quarto.org/docs/books/book-structure.html#references>.

**Reply by NeginValizadegan** [reply]
Thank you!

**Comment by mcanouil** [comment]
You might be interested in the following extension: <https://github.com/pandoc-ext/section-bibliographies>.

**Reply by postylem** [reply]
Isn't this the opposite of what the OP is looking for?

**Reply by mcanouil** [reply]
You are missing the information that it is for a thesis and that I never claimed my reply/information to be the answer especially since one has already been given that addresses the main question.

Also, what is the purpose of your question/comment exactly?

**Reply by postylem** [reply]
sorry! didn't mean it to come across as rude. legitimately was confused. apologies.

**Reply by mcanouil** [reply]
No need to apologise, I was really just asking in case I missed something.

**Reply by postylem** [reply]
To clarify, what I was confused about: I interpreted the OP as primarily asking for a way to collect referenced items from multiples files all together in one bibliography (probably because I was also looking for such), so was wondering if this linked extension offers a way to do that, but I think it can't, though it can do other things that might be useful for a thesis.

**Comment by sorenoneill** [comment]
@krishaamer  .. did you ever get a working solution for this on a quarto _website_ project? ...I need the same thing
